repo/packages/C/cockroachdb/0/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
From the website, "The source code for CockroachDB Enterprise is packaged in the same binary as CockroachDB Core" and that they should contact https://www.cockroachlabs.com/pricing/sales/ 

Anything else to add @a-robinson ?